src/config/header.js

```diff
@@ -2,4 +2,5 @@ export const header = {
   logo: '../images/header/header-logo.svg',
   title: 'ANT Robotics',
   subtitle: 'We simplify employment, team-up, and upgrade of transport robots in warehouses and factories.',
+  curve: '../images/header/header-curve.svg',
```

Comment:
what part is the curve? might there be a way to name it to more clearly insinuate what it is?

Reply:
The curve are the tracks along which the robots move. I'm not sure what would be a better thing to call them.

Reply:
`header__curved-tracks` `animation-tracks` `curved-header-line`

probably fine either way, again just a stylistic thought for practice maintaining a multi-dev codebase (eg. will someone three years from now understand it easily)

Reply:
"will someone three years from now understand it easily" good point)

Reply:
Yeah, those are good suggestions